src/lib/serviceRoutines/getEntityTypes.cpp

```diff
@@ -78,7 +78,11 @@ std::string getEntityTypes
   TIMED_MONGO(mongoEntityTypes(&response, ciP->tenant, ciP->servicePathV, ciP->uriParam, ciP->apiVersion, totalTypesP, true));
 
   std::string rendered;
-  TIMED_RENDER(rendered = response.render(ciP, ""));
+  TIMED_RENDER(rendered = response.render(ciP->apiVersion,
```

Comment:
Lots and lots of very long lines inside TIMED_RENDER.
Easily fixed using help vars.

Just a suggestion ...